homeassistant/components/dsmr_reader/config_flow.py

```diff
@@ -0,0 +1,43 @@
+"""Config flow to configure DSMR Reader."""
+from __future__ import annotations
+
+from collections.abc import Awaitable
+import logging
+from typing import Any
+
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.config_entry_flow import DiscoveryFlowHandler
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def _async_has_devices(_: HomeAssistant) -> bool:
+    """Return true as this integration doesn't support any real devices."""
+    return True
+
+
+class DsmrReaderFlowHandler(DiscoveryFlowHandler[Awaitable[bool]], domain=DOMAIN):
+    """Handle DSMR Reader config flow. The MQTT step is inherited from the parent class."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Set up the config flow."""
+        super().__init__(DOMAIN, "DSMR Reader", _async_has_devices)
+
+    async def async_step_confirm(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Confirm setup."""
+        if user_input is None:
+            return self.async_show_form(
+                step_id="confirm",
+                description_placeholders={
+                    "documentation_link": "https://www.home-assistant.io/integrations/dsmr_reader#setup"
+                },
```

Comment:
We can skip this as the UI now has a documentation link built-in
```suggestion
```
<img width="631" alt="Screen_Shot_2022-08-28_at_4_05_55_PM" src="https://user-images.githubusercontent.com/663432/187094515-6df2d3e9-8d00-49d4-9950-cf965367dbc2.png">
n